website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -69,6 +69,8 @@ Development or Deployment credentials in a dbt project. Simply enter your Snowfl
 username (specifically, the `login_name`) and the corresponding user's Snowflake `password`
 to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowflake user.
 
+**Note**: The schema field in the 'Developer Credentials' section is a required field.
```

Comment:
```suggestion
**Note**: The schema field in the **Developer Credentials** section is a required field.
```

Reply:
it should be bold if it's a UI element (ie these words can be seen in the GUI). if not, leave it as is